benches/Cargo.toml

```
@@ -20,6 +20,10 @@ bevy_render = { path = "../crates/bevy_render" }
 bevy_tasks = { path = "../crates/bevy_tasks" }
 bevy_utils = { path = "../crates/bevy_utils" }
 
+# make bevy_render compile on linux. x11 vs wayland does not matter here as the benches do not actually use a window
+[target.'cfg(target_os = "linux")'.dev-dependencies]
+bevy_internal = { path = "../crates/bevy_internal", features = ["x11"] }
```

```suggestion
bevy_winit = { path = "../crates/bevy_winit", features = ["x11"] }
```

Could you switch this to `bevy_winit` instead of `bevy_internal`? Both do the same thing, but I fear pulling in the entirety of `bevy_internal` may slow down compile times. (We're compiling in release mode!)